azext_edge/edge/_help.py

```diff
@@ -362,6 +362,11 @@ def load_iotops_help():
             data_source={data_source} name={name} observability_mode={observability_mode} sampling_interval={sampling_interval}
             queue_size={queue_size} --data data_source={data_source}
 
+        - name: Create an asset using a file containing data-points and another file containing events.
+          text: >
+            az iot ops asset create --name {asset_name} -g {resource_group} --custom-location {custom_location}
```

Comment:
Suggestion to use more real input examples rather than placeholders, real values can reduce ambiguity on usage, for example `--endpoint myassetendpoint`  clearly conveys an endpoint name.